[IMP] l10n_in_reports_gstr: add suggestion for 30 days duration (#12525)
odoo/documentation: content/applications/finance/fiscal_localizations/india.rst

   .. image:: india/gst-portal-api-yes.png
      :alt: Click Yes

.. note::

>    For the :guilabel:`Duration`, it's recommended to select :guilabel:`30 days` to prevent the need
>    for frequent token reauthentication.

Reviewer: ```suggestion
   It is recommended to set the :guilabel:`Duration` to :guilabel:`30 days` to avoid the need for frequent token reauthentication.
```
